adafruit_wiznet5k/adafruit_wiznet5k_socket.py

```diff
@@ -325,6 +325,26 @@ def recv(self, bufsize=0, flags=0):  # pylint: disable=too-many-branches
         gc.collect()
         return ret
 
+    def embed_recv(self, bufsize=0, flags=0):  # pylint: disable=too-many-branches
```

Comment:
What is this used by?

Reply:
It was excluded because it was a function that was not required for RECV. I put it in for a simple test.